canonical-websites/cloud-init.io
# Migrated the site to Vanilla V1

## Done
Migrated the site to Vanilla V1

## QA
- Pull down code
- Run `/run`
- Check the site looks ok 

## Issue / Card
Fixes https://github.com/ubuntudesign/vanilla-squad/issues/5 

## Demo
http://cloud-init.io-v1.demo.haus/
